conan/tools/cmake/presets.py

```diff
@@ -30,6 +30,10 @@ def _build_preset_name(conanfile):
     return build_type.lower() if build_type else "default"
 
 
+def _test_preset(conanfile, multiconfig):
```

Comment:
So the test preset is basically identical to the build preset, just different name? Maybe would it be possible to do an extension/inclusion of the build one, instead of a repetition (talking from ignorance, I know little about test presets)

Reply:
The [build and test presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#test-preset) share some common fields like `name` and `configurePreset`. But both presets also allow some distinct features. I'll try to do a sensible renaming of the helper functions.